tests/workflows/error_handling_module_path/tests/test_workflow.py

```diff
@@ -0,0 +1,59 @@
+from vellum.workflows.errors.types import WorkflowErrorCode
+
+from tests.workflows.error_handling_module_path.workflow import CustomErrorWorkflow
+
+
+def test_non_vellum_node_error_returns_node_execution_error():
+    workflow = CustomErrorWorkflow()
+
+    terminal_event = workflow.run()
+
+    assert terminal_event.name == "workflow.execution.rejected"
+    assert terminal_event.error.code == WorkflowErrorCode.NODE_EXECUTION
+    assert "An unexpected error occurred while running node" in str(terminal_event.error.message)
+
+
+def test_error_handling_implementation_in_runner():
```

Comment:
delete this test and the test below. we want to keep the tests at the workflow level